documentatie/flowcharts/date-entry-page.md

```diff
@@ -0,0 +1,182 @@
+# Data Entry Page
+
+## Questions
+- Where does "cache input" happen in these flows?
+- Is there a different warning icon for accepted versus not-accepted warnings? Or is it not possible to leave a page without accepting the warning? Also, accepting warnings is not yet included in any of the flows.
+
+
+## Render navigation menu
+
+Render happens based on last received API response.
+
+```mermaid
+flowchart TD
+
+    %% elements
+    flow-start([start])
+    flow-end([end])
+    illegal-state{{illegal state}}
+
+    current-icon-cur-page
+
+    error-icon-next-page
+    %% error-icon-prev-page is illegal state
+
+    warning-icon-next-page
+    warning-icon-prev-page
+
+    error-any-next-page{error-any-next-page?}
+    error-any-prev-page{error-any-prev-page?}
```

Comment:
I think there isn't really a reason to differentiate between previous and next pages from here? Just 'any other' pages should suffice?

also: 'next pages' should be interpreted as 'next pages up to the furthest page the user has seen'. Pages beyond what the user has seen are unreachable, and cannot have an icon

Reply:
updated based on your comment with the text description https://github.com/kiesraad/abacus/commit/02a1e6bd28f4d580e211ea4af9167c2a284cf0fd#commitcomment-145135420